nbs/test/fixtures/lj64/list64.txt

```diff
@@ -0,0 +1,64 @@
+test/fixtures/lj64/wavs/LJ001-0001.wav|Printing, in the only sense with which we are at present concerned, differs from most if not from all the arts and crafts represented in the Exhibition|0
+test/fixtures/lj64/wavs/LJ001-0002.wav|in being comparatively modern.|0
+test/fixtures/lj64/wavs/LJ001-0003.wav|For although the Chinese took impressions from wood blocks engraved in relief for centuries before the woodcutters of the Netherlands, by a similar process|0
+test/fixtures/lj64/wavs/LJ001-0004.wav|produced the block books, which were the immediate predecessors of the true printed book,|0
+test/fixtures/lj64/wavs/LJ001-0005.wav|the invention of movable metal letters in the middle of the fifteenth century may justly be considered as the invention of the art of printing.|0
+test/fixtures/lj64/wavs/LJ001-0006.wav|And it is worth mention in passing that, as an example of fine typography,|0
+test/fixtures/lj64/wavs/LJ001-0007.wav|"""the earliest book printed with movable types, the Gutenberg, or """"forty-two line Bible"""" of about 1455,"""|0
+test/fixtures/lj64/wavs/LJ001-0008.wav|has never been surpassed.|0
+test/fixtures/lj64/wavs/LJ001-0009.wav|Printing, then, for our purpose, may be considered as the art of making books by means of movable types.|0
+test/fixtures/lj64/wavs/LJ001-0010.wav|Now, as all books not primarily intended as picture-books consist principally of types composed to form letterpress,|0
+test/fixtures/lj64/wavs/LJ001-0011.wav|it is of the first importance that the letter used should be fine in form;|0
+test/fixtures/lj64/wavs/LJ001-0012.wav|especially as no more time is occupied, or cost incurred, in casting, setting, or printing beautiful letters|0
+test/fixtures/lj64/wavs/LJ001-0013.wav|than in the same operations with ugly ones.|0
+test/fixtures/lj64/wavs/LJ001-0014.wav|And it was a matter of course that in the Middle Ages, when the craftsmen took care that beautiful form should always be a part of their productions whatever they were,|0
+test/fixtures/lj64/wavs/LJ001-0015.wav|the forms of printed letters should be beautiful, and that their arrangement on the page should be reasonable and a help to the shapeliness of the letters themselves.|0
+test/fixtures/lj64/wavs/LJ001-0016.wav|The Middle Ages brought calligraphy to perfection, and it was natural therefore|0
+test/fixtures/lj64/wavs/LJ001-0017.wav|that the forms of printed letters should follow more or less closely those of the written character, and they followed them very closely.|0
+test/fixtures/lj64/wavs/LJ001-0018.wav|The first books were printed in black letter, i.e. the letter which was a Gothic development of the ancient Roman character,|0
+test/fixtures/lj64/wavs/LJ001-0019.wav|"""and which developed more completely and satisfactorily on the side of the """"lower-case"""" than the capital letters;"""|0
```

Comment:
Why do these files have triple quotes here and other places?

Reply:
It appears the quotes show up in a variety of substitutions.  Many little changes occurred in copying (e.g. fourteen seventy to 1470).... will try copying again